native/CMakeLists.txt

```diff
@@ -2383,13 +2383,10 @@ if(ANDROID)
     )
 
     find_library(LIB_EGL NAMES EGL)
-    find_library(LIB_GLESv2 NAMES GLESv2)
     target_link_libraries(cocos2d PUBLIC
         android
         OpenSLES
         ${LIB_EGL}
-        ${LIB_GLESv2}
-        GLESv2
```

Comment:
What will happen if want to use gles2?

Reply:
Nothing will happen, we load libGLESv2 dynamically